command/env_command_test.go

```diff
@@ -64,6 +64,16 @@ func TestEnv_createAndList(t *testing.T) {
 	defer os.RemoveAll(td)
 	defer testChdir(t, td)()
 
+	// make sure a vars file doesn't interfere
+	err := ioutil.WriteFile(
+		DefaultVarsFilename,
+		[]byte(`foo = "bar"`),
+		0644,
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Oh whoops I misread the test comment thinking you were writing a blank vars file. Looks good!